templates/linkerd.yaml

```diff
@@ -0,0 +1,98 @@
+{{ if .Values.linkerd }}
+apiVersion: helm.cattle.io/v1
+kind: HelmChart
+metadata:
+  name: linkerd-control-plane
+  namespace: kube-system
+spec:
+  bootstrap: true
+  chart: https://helm.linkerd.io/stable/linkerd-control-plane-1.9.6.tgz
+  targetNamespace: linkerd
+  valuesContent: |
+    proxyInjector:
+      namespaceSelector:
+        matchExpressions:
+        - key: config.linkerd.io/admission-webhooks
+          operator: NotIn
+          values:
+          - disabled
+        - key: kubernetes.io/metadata.name
+          operator: NotIn
+          values:
+          - kube-system
+          - cert-manager
+          - aws-identity
+    tolerations:
+    - key: node-role.kubernetes.io/control-plane
+      operator: "Exists"
+    - operator: Exists
+      key: taints.acorn.io/workload
+    nodeselector:
+      servicemesh.acorn.io/cp: true
```

Comment:
What happen if there are two or more worker nodes? This won't guarantee that we can deploy at least one pod to controlplane?

Reply:
we forked the chart.